index.js

```diff
@@ -42,6 +42,9 @@ module.exports = {
     // force no semicolons
     'semi': ['error', 'never'],
 
+    // force one or more spaces between template tag functions and their template literals
+    'template-tag-spacing': ['error', 'always'],
+
```

Comment:
sorry, this isn't the rule I'm looking for... to clarify, in Tram-One, the components look a lot like this
```javascript
html`
  <div>
    <h1>Hello World</h1>
    <SpecialFooter />
  </div>
`
```

What I would like to do is force the indentation in the template to be consistent with the rest of the project (in this case, one tab). In the example above, it would be 1 tab before the starter and ending div tag, and 2 tabs before the h1 and SpecialFooter.

Reply:
Hi @JRJurman : Oh sorry, i mistakenly did that for template functions in js. I will rework on it

Reply:
no worries! I'm actually not sure if there is a rule for this. If there isn't that's fine, I'll close the PR and you'll still get credit for it. Any documentation you can find and put on the issue would be appreciated if that's the case!

Reply:
Hi @JRJurman: Could we use eslint-plugin-html to address this?

Reply:
Hi @JRJurman : How about this: https://eslint.vuejs.org/rules/html-indent.html#rule-details - {
  "vue/html-indent": ["error", type, {
    "attribute": 1,
    "baseIndent": 1,
    "closeBracket": 0,
    "alignAttributesVertically": true,
    "ignores": []
  }]
}

Reply:
it's what we want, but this is for a vue `<template>` tag, Tram-One uses tagged template literal functions.